lua/starfall/libs_sh/entities.lua

```diff
@@ -1026,4 +1072,15 @@ function ents_methods:getCreationTime()
 	return ent:GetCreationTime()
 end
 
+--- Applies a engine effect to the hologram
```

Comment:
Applies should be Checks if

Reply:
Shouldn't it be applied to the entity since it is for all entities and not just holograms?

Reply:
Ya entity, not hologram